config/model_configs/prognostic_edmfx_gcmdriven_fullcolumn.yml

```diff
@@ -0,0 +1,48 @@
+initial_condition: "GCM"
+external_forcing: "GCM"
+external_forcing_file: "/central/groups/esm/zhaoyi/GCMForcedLES/forcing/corrected/HadGEM2-A_amip.2004-2008.07.nc" 
+surface_setup: "GCM"
+turbconv: "prognostic_edmfx"
+implicit_diffusion: true
+implicit_sgs_advection: false
+approximate_linear_solve_iters: 2
+max_newton_iters_ode: 1
+edmfx_upwinding: first_order
+edmfx_entr_model: "Generalized"
+edmfx_detr_model: "Generalized"
+edmfx_sgs_mass_flux: true
+edmfx_sgs_diffusive_flux: true
+edmfx_nh_pressure: true
+edmfx_filter: true
+prognostic_tke: true
+moist: "equil"
+config: "column"
+z_max: 40e3
+z_elem: 60
+z_stretch: true
+dz_bottom: 30
+dz_top: 3000
+perturb_initstate: false
+dt: "10secs"
+t_end: "48hours"
+dt_save_state_to_disk: "6hours"
```

Comment:
may be helpful to add `ode_algo: ARS343` somewhere to ensure we're using the right solver when running with Revise

Reply:
added a line to the yaml file for this (renamed from` _fullcolumn.yml` to `_column.yml`)